staging/nvidia/charts/nvidia-dcgm-exporter/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "nvidia-dcgm-exporter.fullname" . }}-test-connection"
```

Comment:
This should be generated in _helpers.tpl. The maximum name length is 64 characters. fullname is trucated at 64 characters, but `-test-connection` adds another 16 characters making it possible for this to be 80 characters and be rejected by the api.